.infrastructure/14_network.tf

```diff
@@ -0,0 +1,118 @@
+## Network VPC and Security Group for CMS Strapi
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+locals {
+  region   = "eu-south-1"
+  vpc_cidr = "10.0.0.0/16"
+  azs      = slice(data.aws_availability_zones.available.names, 0, 3)
+}
```

Comment:
I would remove them, `region` and `vpc_cidr` are not used, `azs` is used only on vpc

Reply:
change made with last commit